resources/busola-migrator/templates/deployment.yaml

```diff
@@ -59,6 +59,8 @@ spec:
               value: {{ .Values.deployment.env.oidcScope | quote }}
             - name: APP_OIDC_USE_PKCE
               value: {{ .Values.deployment.env.oidcUsePKCE | quote }}
```

Comment:
If we no longer use oidc related values in the business logic I'd remove them from configs.